jsonnet/dashboard.jsonnet

```diff
@@ -18,3 +18,19 @@ local etcdMixin = (import 'github.com/etcd-io/etcd/contrib/mixin/mixin.libsonnet
     'etcd.json': std.manifestJsonEx(etcdMixin.grafanaDashboards['etcd.json'], '    '),
   },
 }
+{
+  apiVersion: 'v1',
+  kind: 'ConfigMap',
+  metadata: {
+    annotations: {
+      'include.release.openshift.io/self-managed-high-availability': 'true',
+      'include.release.openshift.io/single-node-developer': 'true',
+      'release.openshift.io/delete': "true",
+    },
+    labels: {
+      'console.openshift.io/dashboard': 'true',
```

Comment:
There's no need to set `labels` on a deletion manifest, because the CVO will delete the resource if it exists, `labels` isn't part of matching a manifest to a resource (just the group, kind, namespace, and name), and deleted resources don't retain any labels.